.env

```diff
@@ -3,3 +3,4 @@ DB_PASS=squid
 DB_PORT=23798
 PROCESSOR_PROMETHEUS_PORT=3000
 GQL_PORT=4350
+NETWORK=litmus
```

Comment:
I don't think this wants committing :)

Reply:
It is ignored in production and only used locally, agree with your comment below though. I'll do it in another PR, I'm thinking of a script that gets all networks by reading the typegen folder then asks you to choose when running `npm start`